saskatoon/harvest/signals.py

```diff
@@ -49,14 +60,9 @@ def notify_unselected_pickers(sender, instance, **kwargs):
         except ObjectDoesNotExist:
             return
         if instance.status == "Ready" and original_instance.status == "Date-scheduled":
-            unselected_pickers = instance.get_unselected_pickers()
             email_list = list()
             mail_subject = _("[Saskatoon] Request for participation declined")
-            for p in unselected_pickers:
-                picker_name = p.picker.name
-                picker_email = list()
-                picker_email.append(p.picker.email)
-                harvest_details = instance.__str__()
```

Comment:
@tristanlatr note that the string representation for the harvest contains the full name of the property owner and the exact address, e.g.: "Harvest on 23/06/2023 17:25 for Nice Owner at 123 av. Mont-Royal"
Do we really want to send that many details to volunteer pickers that aren't selected for the harvest? Wouldn't it be better to use the public announcement string instead?